### PR TITLE
Add Bigtable

### PR DESCRIPTION
**Pull request by zrosenbauer**
**[https://github.com/zrosenbauer/awesome-bigtable](https://github.com/zrosenbauer/awesome-bigtable)**

**[Bigtable](https://cloud.google.com/bigtable) is a fully managed, scalable NoSQL database service for large analytical and operational workloads, built and managed by Google.**

**Note:** I cannot change the license without direct approval from Google's legal team (so not going to happen).

Pull Requests Reviewed:
1. [https://github.com/sindresorhus/awesome/pull/1794](https://github.com/sindresorhus/awesome/pull/1794)
2. [https://github.com/sindresorhus/awesome/pull/2327](https://github.com/sindresorhus/awesome/pull/2327)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -540,6 +540,7 @@
 - [Apache Spark](https://github.com/awesome-spark/awesome-spark#readme) - Unified engine for large-scale data processing.
 - [Qlik](https://github.com/ambster-public/awesome-qlik#readme) - Business intelligence platform for data visualization, analytics, and reporting apps.
 - [Splunk](https://github.com/sduff/awesome-splunk#readme) - Platform for searching, monitoring, and analyzing structured and unstructured machine-generated big data in real-time.
+- [Google Bigtable](https://github.com/zrosenbauer/awesome-bigtable) - Fully managed wide-column and key-value NoSQL database service on Google Cloud Platform.
```
Comment (aleixmorgadas) [inline readme.md:543]: > The title of your entry should be title-cased and the URL to your list should end in #readme.

**Review by aleixmorgadas — COMMENTED** [review]
Thank you @zrosenbauer for the PR!

I'm reviewing it, and I think you missed some of the guidelines. I will point out the main ones 👍 

> You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).

You need to point in the PR which other PR you reviewed.

> We strongly recommend the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/), but any [Creative Commons license](https://creativecommons.org/choose/) will work.
>
>    Tip: You can quickly add it to your repo by going to this URL: https://github.com/<user>/<repo>/community/license/new?branch=main&template=cc0-1.0 (replace <user> and <repo> accordingly).
>
> A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are WTFPL and [Unlicense](https://unlicense.org/).

I see you're using MIT, please, consider using CC0 License.

----

Those are the main things I saw! I hope it helps! 😄

**Comment by zrosenbauer** [comment]
Also... I forgot to add unicorn 🦄

**Comment by zolagonano** [comment]
@zrosenbauer 

Your awesome list's license shouldn't have anything to do with google and BigTable, because you're not using their software to build your own or anything like that which might require license compatibilities, you're just making a list out of public data and the license of it is only about your list, also the MIT license is a code/software license and its not quite good for licensing other types of content.

Note: The MIT license is not compatible with BigTable's license either, so even if it requires license compatibility, the MIT license wouldn't help.

**Comment by zrosenbauer** [comment]
@zolagonano Employees of Google have contributed to the repository. This means that Google owns the contributions of said employees aka to relicense I need Google's approval (this is from the Bigtable Lead Engineer directly... https://github.com/zrosenbauer/awesome-bigtable/pull/8). If I were to change that, Google wouldn't allow future contributions from those employees (which means that a awesome list about a Google built product can't be contributed to by Google).

The best I could do is say from before date X -> MIT, moving forward CC0 (which is also on the Google banned list).

**Comment by zrosenbauer** [comment]
Google banned OSS License list -> [https://opensource.google/documentation/reference/patching#forbidden](https://opensource.google/documentation/reference/patching#forbidden)

I could look at this one -> [https://creativecommons.org/licenses/by/4.0/](https://creativecommons.org/licenses/by/4.0/)

**Comment by zrosenbauer** [comment]
@sindresorhus any guidance here on the License issue above (I think thats the last piece here of this puzzle)

**Short description of issue**: I originally licensed as MIT (my default) and didn't realize it would conflict with the standards in this repository (I didn't realize the awesome list existed, just saw others in the wild). Google employees contributed to the repository on behalf of Google & I can't switch the license without Google's approval. The best I can do is a stop / start with the new license (recommendation from an engineer there). Also the CC* licenses are apparently banned by Google Legal 😢 ...

If we can resolve this I'd happily help build out a license migration document (unless there is one I'm missing...)

**Comment by sindresorhus** [comment]
> I could look at this one -> https://creativecommons.org/licenses/by/4.0/

This is an acceptable license too.

File: readme.md [hunk 2]
```diff
@@ -540,6 +540,7 @@
 - [Apache Spark](https://github.com/awesome-spark/awesome-spark#readme) - Unified engine for large-scale data processing.
 - [Qlik](https://github.com/ambster-public/awesome-qlik#readme) - Business intelligence platform for data visualization, analytics, and reporting apps.
 - [Splunk](https://github.com/sduff/awesome-splunk#readme) - Platform for searching, monitoring, and analyzing structured and unstructured machine-generated big data in real-time.
+- [Google Bigtable](https://github.com/zrosenbauer/awesome-bigtable#readme) - Fully managed wide-column and key-value NoSQL database service on Google Cloud Platform.
```
Comment (sindresorhus) [inline readme.md:543]:
```suggestion
- [Bigtable](https://github.com/zrosenbauer/awesome-bigtable#readme) - A fully managed, scalable NoSQL database service for large analytical and operational workloads.
```

**Review by andrzejsydor — APPROVED** [review]

**Comment by sindresorhus** [comment]
Closing for lack of activity.

**Pull request closed without merging**